cogs/info.py

```diff
@@ -109,8 +109,8 @@ async def committee_channel_error(self, context, error):
             return True
 
     async def voting_channel_error(self, context, error):
-        if isinstance(error, commands.errors.NoPrivateMessage) or isinstance(error, commands.errors.CheckFailure):
-            await context.send('Please only run this command in public, non-voting channels')
+        if isinstance(error, commands.errors.CheckFailure):
+            await context.send('Please only run this command in DMs, or in a voting channel')
```

Comment:
"in the voting channel", rather than "in a voting channel", as there is only one